frontend/packages/kubevirt-plugin/src/components/vm-disks/guest-agent-file-systems.tsx

```diff
@@ -110,9 +113,18 @@ export const FileSystemsList: React.FC<FileSystemsListProps> = ({ vmi, vmStatusB
 
   return (
     <div className="kubevirt-vm-details__file-systems">
-      <h3>File Systems</h3>
-      The following information regarding how the disks are partitioned is provided by the guest
-      agent.
+      <h3>
+        File Systems
+        <Popover
+          aria-label="File systems description"
+          position="top"
+          bodyContent={<>{GUEST_AGENT_FILE_SYSTEMS_DESCRIPTION}</>}
+        >
+          <Button variant="link" id="kubevirt-vm-details__file-systems--info-icon" isInline>
+            <BlueInfoCircleIcon />
```

Comment:
this should be a grey (?)

Reply:
I can change it. The only concern I have is that this came out of the console-shared package and there is no grey info icon that I can find there. The only place the blue one is used is for the guest agent missing message, but it doesn't appear that it was added for that purpose since it was there before that PR was created.

Do you know if we have a standard for the colors of these icons or somewhere else in console where they're used?

Reply:
I was looking at the Deployments>environment tab and I see the proper icon there.

Reply:
Icon updated!